assets/app/view/create_game.rb

```diff
@@ -45,10 +45,12 @@ def render_content
           }
         end
 
+        inputs << h(:br)
+        inputs << render_button('Upload button') { `document.getElementById('import_json_file').click()` }
```

Comment:
Just `Upload` should be fine ;-)

Reply:
I meant "Upload file" 😂

I added a word, because just "upload" was the same size as "submit" and it somehow looked weird